tests/test_routes.py

```diff
@@ -141,3 +141,19 @@ def test_route_landing_clear_filters(self, flask_test_client):
             assert (
                 b"Assessor dashboard</p>" in response.data
             ), "Response does not contain expected heading"
+
+    @pytest.mark.parametrize(
+        "expected_strings",
+        [b"general-information", b"activities", b"partnerships"],
```

Comment:
Should we check for the names as well as the IDs?

Reply:
good shout, added